### PR TITLE
feat: preparation for 2024 conference on tour

### PR DESCRIPTION
**Pull request by AceTheCreator**
This PR contains the initial requirement for the 2024 conference

### CODE REVIEW

File: config/city-lists.json [hunk 1]
```diff
@@ -1,63 +1,50 @@
...
-		"name": "Madrid",
-		"country": "Spain",
-		"date": "October 19, 2023",
-		"description": "Join us in Madrid for AsyncAPI Conference and learn how to speak fluent API! Let's taco about messaging and have a fiesta you will remember!",
-		"img": "https://images.unsplash.com/photo-1539037116277-4db20889f2d4?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=1470&q=80",
-		"address": "Sngular Madrid, C. de Labastida, 1, 28034 Madrid, Spain",
+		"name": "Helsinki",
+		"country": "Finland",
+		"date": "May 28-29 2024",
+		"description": "Join us in Helsinki for the AsyncAPI Conference and unlock the API of opportunities! Let's bridge the messaging gap and create a tech buzz you will want to experience!",
```
Comment (thulieblack) [inline config/city-lists.json:6]:
```suggestion
		"description": "Join us in Helsinki for the AsyncAPI Conference, where we will explore all things AsyncAPI and connect physical and digital worlds. Get excited as we delve into sustainable APIs for the Era of AI, Data Platforms, and Quantum Computing!.",
```

**Review by thulieblack — CHANGES_REQUESTED** [review]
1. hey @AceTheCreator would it be possible to remove the "AsyncAPI v3 button and replace it with "Call for Speakers Now Open" and redirect the link to the Helsinki Page?
image
2. Make this CfP a bit bigger its hardly noticeable 
image
3. Can we add a paragraph: "We are currently looking for sponsors, for more details please read our Sponsorship Prospectus"
image
4. Finally, you can open up the Call for Speakers/Submit your Proposal, for London and Paris as well. Using the same link

**Comment by thulieblack** [comment]
cc @Mayaleeeee

**Comment by Mayaleeeee** [comment]
> cc @Mayaleeeee 

I'm on it

**Comment by Mayaleeeee** [comment]
- The texts on the location images are difficult to read.

h

Instead, we could have something like the one below; you can check it in the Figma file [here](https://www.figma.com/file/sS5ofLRtuFqDGFvlNBurns/AsyncAPI-Conference-on-Tour'23%2FAACoT?type=design&node-id=1457%3A1002&mode=design&t=MkiKZWZHWuXJFlDO-1).
Paris

**Comment by Mayaleeeee** [comment]
- the distance between the elements in the hero section should be the same length as the [previous year](https://conference.asyncapi.com/). 

Screenshot 2024-02-27 224431

**Comment by AceTheCreator** [comment]
@Mayaleeeee, fantastic suggestions but I'd love to propose them gfi once we merge this PR. wyt?

**Review by thulieblack — APPROVED** [review]

**Pull request merged**